testem.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = {
-  test_page: 'tests/index.html?hidepassed',
+  test_page: 'tests/index.html?hidepassed&fullscreencontainer',
```

Comment:
Because of the failing tests for the dropdown component, I had to add this parameter. This will make sure that tests  are being run without any scaling (normally at 50%) so that correctly positioning the dropdown doesn't break.

For more information see: [this](https://github.com/emberjs/ember-qunit/issues/521) (similar issue) & [this](https://github.com/emberjs/ember-qunit/pull/763) (solution).

Reply:
I looked into this a bit since it also made the tests fail on the `/tests`. It seems this was code by a bug in the clipping detection code of floating ui and it seems to be fixed on the latest 1.1.0 release. So I think we can revert this and simply bump the version.

Reply:
Good find! Did a quick test and it seems to work just fine 👍